packages/database/src/api/Database.ts

```diff
@@ -132,14 +132,13 @@ export class DatabaseInternals {
   constructor(public database: Database) {}
 
   /** @return {Promise<void>} */
-  delete(): Promise<void> {
+  async delete(): Promise<void> {
```

Comment:
How will this be supported in browsers that don't support async/await. Last time I asked - this wasn't being polyfilled.

If it is - that may be a big polyfill.

Reply:
I think TypeScript supports async/await and has downlevel helpers to make the generated code work in ES3/ES5 (https://blog.mariusschulz.com/2016/12/09/typescript-2-1-async-await-for-es3-es5) but we need to see what the resulting code size change is...